### PR TITLE
Refactor/mood page

### PR DESCRIPTION
**Pull request by gjtrecny**
### What (if any) features are you implementing?
- Delete function
### What (if anything) did you refactor?
- Mood alt tags
### Were there any issues that arose?
-

### Refactor checklist
- [X] Check variable and parameter names: are they descriptive and unique?
- [X] Is the code DRY? SRP?
- [X] Is it easy for others to read your code? (explanatory comments needed?)
- [X] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [X] Are commit messages formatted correctly?

### CODE REVIEW

File: src/apiCall.js [hunk 1]
```diff
@@ -28,4 +28,12 @@ const checkResponse = (response) => {
     }
 }
 
-export { getQuote, getUser, updateUser }
\ No newline at end of file
+const deleteJournalEntry = (entryId) => {
```
Comment (anewb87) [inline src/apiCall.js:31]: Awe yeah! Let's delete stuff!

File: src/Components/Card.scss [hunk 2]
```diff
@@ -40,4 +25,34 @@
     background: white;
     width: 10px;
     height: 8px;
+}
+
+
```
Comment (anewb87) [inline src/Components/Card.scss:30]: Like this styling.

**Review by anewb87 — APPROVED** [review]
We can delete!

**Review by tommi-t-nguyen — APPROVED** [review]
👍

**Pull request merged**